spec.html

```diff
@@ -377,7 +478,8 @@ <h1>InnerModuleEvaluation( _module_, _stack_, _index_ )</h1>
             1. Remove the last element of _stack_.
             1. Set _requiredModule_.[[Status]] to `"evaluated"`.
             1. If _requiredModule_ and _module_ are the same Module Record, set _done_ to *true*.
-            1. <ins>Otherwise, set _requiredModule_.[[ExecPromise]] to _module_.[[ExecPromise]].</ins>
+            1. <ins>Otherwise, if _module_.[[Async]] is *true*,</ins>
```

Comment:
I know it's unrelated to this PR, but it might be worth adding a note in this spec for this line that this code path ensures cyles have a single execution promise, which I think is an important concept?

Reply:
Yes, that's a good point. This is rather subtle logic. (Do you agree with it?)

Reply:
Yes, I completely agree with it, and think this is a crucial feature for TLA in cycles, that is worth noting more clearly.